src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27`` Retina 5K Monoblock"
+        class="card__image"
+      >
+
+      <h3 class="card__logo">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <span class="card__code">Product code: 195434</span>
+
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <span class="card__review-text">Reviews: 5</span>
+      </div>
+
+      <div class="card__price">
+        <span class="card__price-text">Price:</span>
+        <span class="card__price-number">$2,199</span>
+      </div>
+
+      <a class="card__buy" data-qa="hover">
+        Buy
+      </a>
+    </div>
+
+
   </body>
```

Comment:
```suggestion
    </div>
  </body>
```
Don't leave empty lines between parent and child elements